### PR TITLE
{lang}[foss/2017b,intel/2017b] cysignals v1.10.2, Cython v0.29.10, gmpy2 v2.1.0b1, ... w/ Python 2.7.14 + 3.6.3

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8a1a900e4e1b485d213c9a3f61f4f7ee for a full test report.

File: easybuild/easyconfigs/p/ppl/ppl-1.2-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('GMP', '6.1.2'),
+]
+
+sanity_check_paths = {
+    'files': ["bin/ppl-config", "bin/ppl_pips", "lib/libppl.so", "include/ppl.hh"],
```
[the remark this answers is not in the document]
Comment (murfreesboro) [reply easybuild/easyconfigs/p/ppl/ppl-1.2-GCCcore-6.4.0.eb]: used the SHLIB_EXT, and changed the checking for dirs.

[the remark this answers is not in the document]
Comment (murfreesboro) [reply easybuild/easyconfigs/p/ppl/ppl-1.2-GCCcore-6.4.0.eb]: yes the much newer Cython is needed for the cysignals package, now I added comments in the cysignals ebconfig file.

File: easybuild/easyconfigs/p/ppl/ppl-1.2-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('GMP', '6.1.2'),
+]
+
+sanity_check_paths = {
+    'files': ["bin/ppl-config", "bin/ppl_pips", "lib/libppl.%s' % SHLIB_EXT", "include/ppl.hh"],
```
[the remark this answers is not in the document]
Comment (murfreesboro) [reply easybuild/easyconfigs/p/ppl/ppl-1.2-GCCcore-6.4.0.eb]: OK Thank you @boegel for the change!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
node3139.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d3fe74426262175772ae2d1248c66543 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
node2419.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ed9e2dd3b9ba6e4d5fab4e8afb5128ca for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 17 out of 17 (17 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b208c06641420f74aee4f5c5af190b51 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @murfreesboro!

**Pull request merged**